Formula/yaml.rb

```diff
@@ -0,0 +1,30 @@
+class Yaml < Formula
+  desc "Process YAML documents from the CLI"
+  homepage "https://github.com/mikefarah/yaml"
+  url "https://github.com/mikefarah/yaml/archive/1.13.1.tar.gz"
+  sha256 "5512c586c6dd4f9b838d80facde656153b4fe3c1ce4992cc3057f0930fec4c7a"
+
+  depends_on "go" => :build
+  depends_on "govendor" => :build
+
+  def install
+    contents = Dir["{*,.git,.gitignore}"]
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/mikefarah/yaml").install contents
+
+    ENV["GOPATH"] = gopath
+    ENV.prepend_create_path "PATH", gopath/"bin"
+
+    cd gopath/"src/github.com/mikefarah/yaml" do
+      system "govendor", "sync"
+      system "go", "build", "-o", "bin/yaml"
```

Comment:
can we not do `bin/"yaml"` directly?

Reply:
Sorry - haven't completed wrapped my head around how homebrew works - I was going off another Go formula in here (stern). 

Not entirely sure I understand what these comments are implying :/ I'll give it a go though

Reply:
Not sure what you mean by do bin/yaml directly?

Reply:
Instead of `"-o", "bin/yaml"` use `"-o", bin/"yaml"`.

Reply:
(And then ditch the `bin.install` line below).